File: Patterns/hollowRectangle.js
for(let row = 0; row < 6; row++){
    for(let col = 0; col < 5; col++){
        if(row === 0 || row === 5){
            process.stdout.write("* ")
        }else {
            if(col === 0 || col === 4){
                process.stdout.write("* ")
            } else {
                process.stdout.write("  ")
            }
        }
    }
    // process.stdout.write("\\n")
    console.log()
}

File: Patterns/hollowRectangle.cpp
#include<iostream>
using namespace std;

int main(){
    int row;
    cout << "Enter no of rows : ";
    cin >> row;
    cout << endl;
    int col;
    cout << "Enter no of cols : ";
    cin >> col;

    for(int i=0; i<row; i++){
        for(int j=0; j<col; j++){
            if(i == 0 || i == row-1 || j == 0 || j == col-1){
                cout << "* ";
            }else{
                cout << "  ";
            }
        }
        cout<< endl;
    }

    return 0;
}